compileProtobuffers.ts

```diff
@@ -0,0 +1,42 @@
+import fs from "fs-extra";
+import { exec } from "node:child_process";
+
+const PROTOBUFF_DIR = "./Ecdar-ProtoBuf/";
+const OUT_DIR = "./src/lib/proto/";
+
+export const compileProtobuffers = {
+	name: "Compiling protobuffers",
+
+	buildStart: async () => {
+		await Promise.all(
+			(await fs.readdir(PROTOBUFF_DIR))
+				.filter((file) => file.match(/.*\.proto/g))
+				.map((file) =>
+					runcmd(`
+					  npx protoc \\
+						  --ts_out ${OUT_DIR} \\
+						  --proto_path ${PROTOBUFF_DIR} \\
+						  ${PROTOBUFF_DIR}/${file}
+					`),
+				),
+		);
+		console.log("\x1b[33m✔\x1b[0m Compiled Protobuffers Done");
```

Comment:
Adding colors is cool, but also very unreadable in the code and in build terminals, where these colors are not understood.

I won't force you to fix it, but I would prefer it if you remove the colors, or use a library that only adds colors in terminals that support it. I have used and can recommend: https://www.npmjs.com/package/chalk